handler.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+from pathlib import Path
+
+import boto3
+import redis
+
+from s3_log_shipper.parsers import ParserManager
+from s3_log_shipper.s3_event_models import S3Event
+from s3_log_shipper.shipper import RedisLogShipper
+
+log: logging.Logger = logging.getLogger(__name__)
+
+
+def get_log_parser() -> ParserManager:
+    return ParserManager(config_file=get_config_file())
+
+
+def get_config_file() -> Path:
+    default = f"{os.path.dirname(__file__)}/input_files.json"
+    path = os.environ.get("CONFIG_FILE", default)
+    file = Path(path)
+    if not file.exists() or not file.is_file():
+        msg = f"Expected to find parser config file at {file.absolute()}. No such file found."
+        log.error(msg)
+        raise Exception(msg)
+    return file
+
+
+def get_output_redis_host_from_environment():
+    try:
+        return os.environ["REDIS_HOST"]
+    except KeyError as e:
+        raise Exception("env variable is not found: {}".format(e))
+
+
+def get_output_redis_port_from_environment():
+    try:
+        return os.environ["REDIS_PORT"]
+    except KeyError as e:
+        raise Exception("env variable is not found: {}".format(e))
+
+
+S3_CLIENT = boto3.client("s3")
```

Comment:
This _may_ be a bit of overkill, but do you mind injecting in these values rather than having globals?

Reply:
I'm afraid this is done this way in line with the AWS Lambda guidelines, in order to keep these variables cached in the event that the lambda is kept warm.